sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -131,11 +143,7 @@ public void error(String format, Object... args) {
      * @param args Arguments for the message, if an exception is being logged last argument is the throwable.
      */
     private void log(LogLevel logLevel, String format, Object... args) {
-        LogLevel environmentLoggingLevel = getEnvironmentLoggingLevel();
-
-        if (canLogAtLevel(logLevel, environmentLoggingLevel)) {
-            performLogging(logLevel, environmentLoggingLevel, false, format, args);
-        }
+        performLogging(logLevel, false, format, args);
```

Comment:
Could this method be removed and all the higher level methods just call into `performLogging`? This previously existed to abstract a few pieces of logic away from the public API methods which no longer exists based on the changes made in this PR.

Reply:
Sure.